xbmc/pvr/PVRGUIActions.cpp

```diff
@@ -700,6 +702,50 @@ namespace PVR
                                             CVariant{timer->Title()});
   }
 
+  class AsyncRecordingAction : private IRunnable
+  {
+  public:
+    bool Execute(const CFileItemPtr &item = CFileItemPtr());
```

Comment:
Not sure whether is more a theoretical problem or something for people not knowing about the "unsafeness" here. But I see the performance impact and to some extend this makes it senseful to use refs.